<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>39</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>RTKLIB</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="2">
    <widget class="QPushButton" name="BtnStr">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>STRSVR</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rtkget_qt/rtkget_qt.qrc">
       <normaloff>:/icons/rtk6.bmp</normaloff>:/icons/rtk6.bmp</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="BtnNavi">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>RTKNAVI</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rtkget_qt/rtkget_qt.qrc">
       <normaloff>:/icons/rtk5.bmp</normaloff>:/icons/rtk5.bmp</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="BtnConv">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>RTKCONV</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rtkget_qt/rtkget_qt.qrc">
       <normaloff>:/icons/rtk3.bmp</normaloff>:/icons/rtk3.bmp</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="BtnPlot">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>RTKPLOT</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rtkget_qt/rtkget_qt.qrc">
       <normaloff>:/icons/rtk2.bmp</normaloff>:/icons/rtk2.bmp</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="BtnPost">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>RTKPOST</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rtkget_qt/rtkget_qt.qrc">
       <normaloff>:/icons/rtk1.bmp</normaloff>:/icons/rtk1.bmp</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="BtnNtrip">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>NTRIP Browser</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rtkget_qt/rtkget_qt.qrc">
       <normaloff>:/icons/rtk7.bmp</normaloff>:/icons/rtk7.bmp</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="BtnGet">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>RTKGET</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rtkget_qt/rtkget_qt.qrc">
       <normaloff>:/icons/rtk8.bmp</normaloff>:/icons/rtk8.bmp</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QPushButton" name="BtnTray">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>15</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Task Tray</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../rtkget_qt/rtkget_qt.qrc">
       <normaloff>:/buttons/sqr.bmp</normaloff>:/buttons/sqr.bmp</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rtkget_qt/rtkget_qt.qrc"/>
 </resources>
 <connections/>
</ui>
